crates/holochain/src/core/ribosome/host_fn/get_details.rs

```diff
@@ -215,13 +215,10 @@ pub mod slow_tests {
         holochain_trace::test_run();
         // agents should not pass around data
         let config = SweetConductorConfig::rendezvous(false)
-            /*
-            .tune(|config| {
-                config.disable_historical_gossip = true;
-                config.disable_recent_gossip = true;
-                config.disable_publish = true;
+            .tune_network_config(|nc| {
```

Comment:
More commented code, the test wasn't really valid without these being restored. Sure it was passing, but passing correctly now